build.gradle

```diff
@@ -970,6 +970,15 @@ project('test:system') {
         ext.dockerRegistryUrl = project.hasProperty("dockerRegistryUrl") ? project.dockerRegistryUrl : ""
         ext.repoUrl = project.hasProperty("repoUrl") ? project.repoUrl : ""
 
+        def excludedTests = System.getProperty('excludeSystemTests')
+        def includedTests = System.getProperty('includeSystemTests')
+        // Let included tests take precedence if both are specified.
+        if (includedTests != null) {
+            includes = includedTests.split(',').collect { getSystemTestClass(it) }
+        } else if (excludedTests != null ) {
+            excludes = excludedTests.split(',').collect { getSystemTestClass(it) }
+        }
```

Comment:
There are two reasons. After some experimentation I've concluded that due to how the test framework is setup, we are unable to leverage the `--tests` flag / `test.filter` functionality. It works at the class level, but breaks down once you specify the method to use. 

The nice thing about the `TestFilter` class is that it provides much of the work I've manually implemented with `getSystemTestClass`. The downside is that there is no 'exclude' filter, it is seemingly only inclusive. Having an option to exclude tests makes certain use cases much easier, so we still want that. Because of this, I see no point of using `TestFilter` functionality if we cannot isolate specific methods.